src/ipcache.cc

```diff
@@ -702,7 +702,7 @@ ipcache_init(void)
     ipcache_low = (long) (((float) Config.ipcache.size *
                            (float) Config.ipcache.low) / (float) 100);
     n = hashPrime(ipcache_high / 4);
-    ip_table = hash_create((HASHCMP *) strcmp, n, hash4);
+    ip_table = hash_create((HASHCMP *) strcasecmp, n, hash4);
 
```

Comment:
Sorry, I forgot to mention that we should remove a Tolower() call in ipcache_entry constructor because our hash table is case-insensitive now. That removal will save a few CPU cycles, partially offsetting the additional expense of computing a case-insensitive hash on every lookup.

I know that you have given me a permission to modify the PR branch (thank you!). I just do not want to interfere with your ongoing/concurrent modifications. If you want me to do any changes, just ping me.